core/templates/services/html-length.service.ts

```diff
@@ -113,12 +138,22 @@ export class HtmlLengthService {
   }
 
   /**
-   * Calculates the weight of text content based on the calculation type.
-   * @param {string} textContent - The text content to calculate weight for.
-   * @param {CalculationType} calculationType - The calculation
-   * type ('word' or 'character').
-   * @returns {number} The weight of the text content.
-   */
+    * This function calculates the length of a given text content
+    * based on the specified calculation type.
+    *
+    * @param {string} textContent - The text content for which the
+    *    length is to be calculated. This is typically a string of
+    *    text that may include words, spaces, punctuation, and other
+    *    characters.
+    * @param {CalculationType} calculationType - The method used to
+    *    calculate the length. It can be either 'word' or 'character'.
+    *    If 'word', the function counts the number of words in the
+    *    text content. If 'character', it counts the number of
+    *    characters.
+    * @returns {number} The length of the text content. This is
+    *    calculated as the number of words or characters in the text
+    *    content, depending on the calculation type.
+    */
 
   private calculateTextWeight(
```

Comment:
You are still using "weight" here synonymously with "length" in other places.

Are these two concepts the same? If not, what is the difference between weight/length exactly? Try to make sure that you have a consistent definition for each word, so that it's clear where it should be "weight" and where it should be "length".

Please fix elsewhere too, as needed.

Reply:
Weight and length both specify the same concept, in the last comment I got confused that you were just asking for the comments part. Now  'Length has been standardized as people can get confused with weight. Done!